### DISCUSSION TITLE
csv带引号的数据导入支持

### DISCUSSION THREAD

**Discussion by hwfbest** [apache/doris#12399] [RFC]
"ID","PROV_NAME","CITY_CODE","CITY_NAME","KEY_WORD","RESULT_NAME","ADDRESS","TEL","TOTAL_NUM","IN_DATE"
"85","安徽","188","亳州市","谯东镇广告","文化传媒有限公司","高新区嵇康路万达广场","无联系电话","137","2022/8/16 21:19:49"

**Comment by JNSimba** [comment]
This has to be dealt with now, remove the leading and trailing `"`, and then use `","` as the delimiter.

**Reply by hwfbest** [reply]
数据字段内容中存在","，会带来数据的异常

**Reply by hopepanwei** [reply]
希望能把csv中的空导入为NULL，而不是空字符

**Reply by raykimurayyz** [reply]
字段中有json字符串，这样的话，很多分隔符就无法使用了

**Reply by JNSimba** [reply]
> 希望能把csv中的空导入为NULL，而不是空字符

@hopepanwei \N in csv is NULL

**Reply by JNSimba** [reply]
> 字段中有json字符串，这样的话，很多分隔符就无法使用了

@mickytmnew Consider using hidden characters

**Reply by JNSimba** [reply]
> 数据字段内容中存在","，会带来数据的异常

@hwfbest  I think the `"` in the data will be escaped as `\"` ?

**Reply by hopepanwei** [reply]
> > 希望能把csv中的空导入为NULL，而不是空字符
> 
> @hopepanwei \N in csv is NULL

嗯，我在文档中找到了相关说明，但是希望能把这项改为配置，比如http请求头，毕竟上百gb的csv导出如果加上字符串处理会相当的慢
